src/main/io/ledstrip.c

```diff
@@ -980,6 +983,11 @@ void pgResetFn_specialColors(specialColorIndexes_t *instance)
     memcpy_fn(instance, &defaultSpecialColors, sizeof(defaultSpecialColors));
 }
 
+void pgResetFn_ledType(ledType_t *instance)
+{
+    defaultLedType.t = RGB;
+    *instance = defaultLedType;
```

Comment:
http://stackoverflow.com/questions/13415321/difference-between-static-auto-global-and-local-variable-in-the-context-of-c-a
```
ledType_t def = { .t = RGB };
*instance = def;
```